### PR TITLE
Add form-field

### PR DESCRIPTION
**Pull request by wytrych**

### CODE REVIEW

**Comment by wytrych** [comment]
@incuna/js @incuna/frontend Please review

**Comment by wytrych** [comment]
@incuna/js @incuna/frontend  Fixed

**Comment by henrahmagix** [comment]
@wytrych Could you squash the "testing and fixing things in travis" commits please? Maybe squash down to include the linting and small fixes commits

**Comment by wytrych** [comment]
@henrahmagix Done

File: src/init.js [hunk 1]
```diff
@@ -4,7 +4,8 @@
     angular.module('djangularRestFramework', [
         'drf-config',
         'drf-provider',
-        'drf-display-name-filter'
+        'drf-display-name-filter',
+        'drf-field-errors'
```
Comment (henrahmagix) [inline src/init.js]: Doesn't the `drf-form-field` module need to be depended-on here?

**Comment by henrahmagix** [comment]
👍

**Pull request merged**